packages/six.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Setuptools < Package
+  description 'Six is a Python 2 and 3 compatibility library.'
+  homepage 'https://github.com/benjaminp/six'
+  version '1.11.0'
+  source_url 'https://github.com/skycocker/chromebrew/raw/cc4f47007b6fe702e03de28347e20d3a0bb692f6/install.sh'
```

Comment:
Why don't you grab the source from `https://github.com/benjaminp/six/archive/1.11.0.tar.gz` and run the setup.py script?

Reply:
Sure.